SPECS/kernel/kernel.spec

```diff
@@ -42,6 +42,7 @@ Source3:        sha512hmac-openssl.sh
 Source4:        cbl-mariner-ca-20211013.pem
 Source5:        cpupower
 Source6:        cpupower.service
+Patch0:		0001-add-mstflint-kernel-4.28.patch
```

Comment:
Can we entangle the version of the patch with the version of the mstflint package? This will help future devs remember that these versions must be aligned. Including the patch version would be best, too, not just 4.28, but 4.28.0.

Reply:
You can define mstflintver as a global variable in both kernel.spec and mstflint.spec, actually use them in the spec, then entangle the values of these two global variables like sdkver in check_entangled_specs.py

Reply:
https://github.com/microsoft/azurelinux/blob/3.0-dev/toolkit/scripts/check_entangled_specs.py